src/xdp/trn_agent_xdp.c

```diff
@@ -410,29 +422,29 @@ static __inline int trn_process_inner_ip(struct transit_packet *pkt)
 			__u8 conn_allowed = (pkt->inner_ipv4_tuple.protocol == IPPROTO_UDP) ? FLAG_REEVAL : 0;
 			conntrack_set_conn_state(&conn_track_cache, pkt->agent_ep_tunid, &pkt->inner_ipv4_tuple, conn_allowed);
 		}
-	}
 
-	/* Check if we need to apply a forward flow update */
+		/* Check if we need to apply a forward flow update */
```

Comment:
Can you please fix this so tab use is consistent?

Reply:
This is for tab consistency.